### PR TITLE
added system requirement

### PR DESCRIPTION
**Pull request by Abhi-gits**

### CODE REVIEW

File: version 2.0/getSize.c [hunk 1]
```diff
@@ -79,7 +78,7 @@ void getSize(const char *filename, int foodcount)
     }
 
     fclose(in_file);
-    // Closing file
```
Comment (anurag629) [inline version 2.0/getSize.c:82]: do not remove this

File: version 2.0/getSize.c [hunk 2]
```diff
@@ -104,21 +103,7 @@ void getSize(const char *filename, int foodcount)
...
-    // for (int j = 1; j < snakeSize; j++)
-    // {
-    //     TwoDBox[1][j] = '-';
-    // }
-    // for (int j = 0; j < snakeSize; j++)
-    // {
-    //     snakePosition[j] = 1 * c + j + 1;
-    // }
-
-    // start reading
```
Comment (anurag629) [inline version 2.0/getSize.c:121]: do not remove this

File: version 2.0/getSize.c [hunk 3]
```diff
@@ -179,21 +164,7 @@ void getSize(const char *filename, int foodcount)
     }
 
     fclose(in_filee);
-    // End Reading file
```
Comment (anurag629) [inline version 2.0/getSize.c:182]: do not remove this

**Review by anurag629 — CHANGES_REQUESTED** [review]

File: version 2.0/getSize.c (hunk 2 above)
Comment (Abhi-gits) [reply version 2.0/getSize.c:121]: done

File: version 2.0/getSize.c (hunk 3 above)
Comment (Abhi-gits) [reply version 2.0/getSize.c:182]: done

File: version 2.0/getSize.c (hunk 1 above)
Comment (Abhi-gits) [reply version 2.0/getSize.c:82]: done

**Review by anurag629 — APPROVED** [review]

**Pull request merged**